### PR TITLE
chore: adapt pep

### PR DESCRIPTION
**Pull request by keita1523**

### CODE REVIEW

File: src/caret_analyze/common/util.py [hunk 1]
```diff
@@ -143,8 +150,14 @@ def find_similar_one_multi_keys(
...
-        conditions : Callable[[Any], Dict[str, str]]
-        items : Collection[Any]
+        target_names: Dict[str, str]
+            target_names
+        items: Collection[Any]
+            items
+        keys: Callable[[Any], Dict@str, str]]
+            key
+        th: float
+            threshold
```
Comment (hsgwa) [inline src/caret_analyze/common/util.py:160]:
```suggestion
            Similarity judgment threshhold.
            A candidate is mentioned only if it is higher than the threshold.
```

File: src/caret_analyze/common/util.py [hunk 2]
```diff
@@ -143,8 +150,14 @@ def find_similar_one_multi_keys(
 
         Parameters
         ----------
-        conditions : Callable[[Any], Dict[str, str]]
-        items : Collection[Any]
+        target_names: Dict[str, str]
+            target_names
+        items: Collection[Any]
+            items
+        keys: Callable[[Any], Dict@str, str]]
```
Comment (hsgwa) [inline src/caret_analyze/common/util.py:157]:
```suggestion
        keys: Callable[[Any], Dict[str, str]]
```

File: src/caret_analyze/common/util.py [hunk 3]
```diff
@@ -98,8 +99,14 @@ def find_similar_one(target_name: str,
...
-        condition : Callable[[Any], str]
-        items : Collection[Any]
+        target_name: str
+            target_name
+        items: Collection[Any]
+            items
+        key: Callable[[Any], str]
+            key
+        th: float
+            threshold
```
Comment (hsgwa) [inline src/caret_analyze/common/util.py:109]:
```suggestion
            Similarity judgment threshhold.
            A candidate is mentioned only if it is higher than the threshold.
```

File: src/caret_analyze/common/util.py [hunk 4]
```diff
@@ -61,8 +61,9 @@ def find_one(condition: Callable[[Any], bool], items: Optional[Iterable[Any]]) -
         Parameters
         ----------
         condition : Callable[[Any], bool]
+            condition
         items : Optional[Iterable[Any]]
-
+            items
```
Comment (hsgwa) [inline src/caret_analyze/common/util.py:66]:
```suggestion
            Items to be searched.
```

**Review by hsgwa — CHANGES_REQUESTED** [review]
One more warning is left in latest main branch.
```
src/caret_analyze/infra/lttng/ros2_tracing/data_model_service.py:171 in private method `_ensure_dataframe`: D416: Section name should end with a colon ('Note:', not 'Note')
```

File: src/caret_analyze/common/util.py (hunk 1 above)
Comment (keita1523) [reply src/caret_analyze/common/util.py]: Changed sentence.

File: src/caret_analyze/common/util.py (hunk 3 above)
Comment (keita1523) [reply src/caret_analyze/common/util.py]: Changed sentence as your comments

**Comment by keita1523** [comment]
Adding the word `TILDE` to dictionary.
https://github.com/tier4/cspell-dicts/pull/12

**Review by hsgwa — APPROVED** [review]
LGTM

**Pull request merged**